packages/common-common/src/statsd.ts

```diff
@@ -0,0 +1,31 @@
+import { StatsD } from 'hot-shots';
+
+export enum StatsDTag {
+  Commonwealth = "commonwealth",
+  TokenBalanceCache = "token-balance-cache",
+}
+
+// Singleton containing map of tag to StatsD instance, allows us to keep track of stats for different components
+export class StatsDController {
+  private static instance: StatsDController = new StatsDController();
+
+  private _client: Map<StatsDTag, StatsD>;
+
+  private constructor() {
+    // Create map each of the StatsDTag enum to its own StatsD instance.
+    this._client = new Map(Object.values(StatsDTag).map((key: StatsDTag) => [key, this.createStatsD(key)]));
```

Comment:
Why do we need to use a map here? One instance should be fine so long as we vary the tags.

Reply:
My thinking was that since we are using a singleton pattern, instead of writing project tags each time we send the stat, we could just do it through having singletons for each statsd object. But having one instance is fine too.